app/config/publications/overview-table-columns.js

```diff
@@ -1,4 +1,25 @@
 export default [
+  {
+    keyName: 'speedProcedure',
+    translationKey: 'publications-table-speed-procedure',
+    translationKeySmall: 'publications-table-speed-procedure-small',
+    sortKey: 'urgency-level.position',
+    showByDefault: true,
+  },
+  {
+    keyName: 'publicationNumber',
+    translationKey: 'publications-table-publication-number',
+    translationKeySmall: 'publications-table-publication-number-small',
+    showByDefault: true,
+    sortKey: 'identification.structured-identifier.local-identifier,-created',
+  },
+  {
+    keyName: 'numacNumber',
+    translationKey: 'publications-table-numacnummer-bs',
+    translationKeySmall: 'publications-table-numacnummer-bs-small',
+    sortKey: 'numac-numbers.id-name,-created',
```

Comment:
Sortering zou hier ook rekening kunnen houden met het versienummer.

Reply:
Ik vermoed dat je deze comment bij `publicationNumber` bedoelde ipv bij `numacNumber` aangezien er voor numac nummer geen versie ingevuld wordt?

Het veld met de versie is een string. Daarop sorteren geeft niet het gewenste effect voor cijfers of voor termen als 'bis', 'tris', ... De 2e sortering op `-created` is daarom toegevoegd in de veronderstelling dat nieuwere versies later gemaakt worden. Dat is zeker niet foutloos, maar een best effort.